tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -296,15 +335,20 @@ def test_snowflake(
             param(
                 "snowflake", get_random_identifier_name(), marks=[pytest.mark.snowflake]
             ),
+            param(
+                "databricks_sql",
+                PYTHON_VERSION,
+                marks=[pytest.mark.databricks],
```

Comment:
This particular test will continue to fail until I make the changes to how the `gx ExecutionEngine` is created.